src/commonMain/kotlin/org/kosat/CDCL.kt

```diff
@@ -73,7 +73,7 @@ class CDCL(private val solverType: SolverType = SolverType.INCREMENTAL) : Increm
     data class VarState(
         var status: VarStatus,
         var clause: Clause?,
-        var level: Int,
+        var level: Int
```

Comment:
It was better with a [trailing comma](https://kotlinlang.org/docs/coding-conventions.html#trailing-commas) :)